contrib/labstack/echo/echotrace_test.go

```diff
@@ -0,0 +1,126 @@
+package echo
+
+import (
+	"errors"
+	"net/http/httptest"
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/labstack/echo"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestChildSpan(t *testing.T) {
+	assert := assert.New(t)
+	mt := mocktracer.Start()
+	defer mt.Stop()
+
+	router := echo.New()
+	router.Use(Middleware("foobar"))
+	router.GET("/user/:id", func(c echo.Context) error {
+		_, ok := tracer.SpanFromContext(c.Request().Context())
+		assert.True(ok)
+		return nil
```

Comment:
We should also assert that this handler was called. Let's add a boolean at the top of the test:
```go
var called bool
```
Then, inside the handler:
```go
called := true
```
...and at the very end of the test after serving:
```go
assert.True(called)
```

Reply:
That is good idea. See other comments, please.